### PR TITLE
Reset MiMa to use 2.13.0-RC2 artifacts

### PR DESCRIPTION
**Pull request by ashawley**

### CODE REVIEW

File: build.sbt [hunk 1]
```diff
@@ -83,63 +83,11 @@ ThisBuild / headerLicense  := Some(HeaderLicense.Custom(
...
 
-// to be locked down sometime around the time of 2.13.0-RC1
-Global / mimaReferenceVersion := Some("2.13.0-RC1")
+Global / mimaReferenceVersion := Some("2.13.0-RC2")
 
-// Drop once 2.13.0 is out.
 import com.typesafe.tools.mima.core._
 val mimaPrereleaseHandlingSettings = Seq(
   mimaBinaryIssueFilters ++= Seq(
-    ProblemFilters.exclude[Problem]("scala.reflect.internal.*"),
```
Comment (dwijnand) [inline build.sbt]: Keep this one.

**Review by dwijnand — CHANGES_REQUESTED** [review]
Nice one, thanks!  Just 1 change.

**Review by dwijnand — DISMISSED** [review]

File: versions.properties [hunk 2]
```diff
@@ -1,5 +1,5 @@
 # Scala version used for bootstrapping (see README.md)
-starr.version=2.13.0-RC1
+starr.version=2.13.0-RC2
```
Comment (dwijnand) [inline versions.properties:2]: I don't know if this is right to change.  (Seems definitely out of scope.)

Comment (adriaanm) [reply versions.properties:2]: Yes, now is the right time to bump starr. Usually we do this in a separate commit, but nbd.

**Pull request merged**